blazor/media-query/getting-started.md

```diff
@@ -0,0 +1,447 @@
+---
+layout: post
+title: Getting Started with Blazor Media Query Component | Syncfusion
+description: Checkout and learn about getting started with Blazor Media Query component in Blazor Server App and Blazor WebAssembly App.
+platform: Blazor
+control: Media Query
+documentation: ug
+---
+
+# Getting Started with Blazor Media Query Component
+
+This section briefly explains about how to include `Blazor Media Query` component in your Blazor Server App and Blazor WebAssembly App using Visual Studio. 
+
+## Prerequisites
+
+* [System requirements for Blazor components](https://blazor.syncfusion.com/documentation/system-requirements)
+
+## Create a new Blazor App in Visual Studio
+
+You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+
+* [Create a Project using Microsoft Templates](https://docs.microsoft.com/en-us/aspnet/core/blazor/tooling?pivots=windows)
+
+* [Create a Project using Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-code-integration/create-project)
+
+## Install Syncfusion Blazor Layouts NuGet in the App
```

Comment:
Syncfusion.Blazor.Core package, Layouts package not contained media query.

Reply:
Hi @silambarasani ,

I modified the content as per discussion.

Regards,
@Backiaraj